redi-connect-backend/lib/build-backend-url.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
       return 'https://api.staging.connect.redi-school.org'
     } else if (env === 'development') {
       return 'http://127.0.0.1:3003'
+    } else if (env === 'prod') {
+      return 'https://app-staging-redi.azurewebsites.net'
```

Comment:
Ouch, I don't like how subtle the difference here is between `production` meaning AWS and `prod` meaning Azure. If it's quick to rename `prod` into something more explicit like `prod:azure`, can we do that?

Reply:
If not, we can keep `prod`, but we really have to kill AWS as soon as possible, and make everyone else aware of what is what here. This can quickly lead to a slippery slope down undeclarative confusion between `prod` and `production`.

Reply:
wait. this is a previous commit. I was testing something but then I completely removed the `prod` related code